tests/accuracy/test_global_correction.py

```diff
@@ -195,3 +244,52 @@ def _create_fake_measurement(tmp_path, model_path, twiss_path, error_val, optics
             meas[ERROR] = meas.loc[:, f"{ERR}{col}"].to_numpy()
         meas[WEIGHT] = 1.
     return twiss_df, model_df, meas_dict
+
+# Is the following better placed in a folder with reference files, 
+# or to be generated on the fly?
+from omc3 import madx_wrapper
+from omc3.optics_measurements.constants import PHASE_ADV
+def run_dpp(tmp_path, offset, beam):
+    Qx = 62.28001034
+    Qy = 60.31000965
+    script = f"""
+    option, -echo;
+    call, file = 'omc3/model/madx_macros/general.macros.madx';
+    call, file = 'omc3/model/madx_macros/lhc.macros.madx';
+    call, file = 'omc3/model/accelerators/lhc/2018/main.seq';
+    option, echo;
+    exec, cycle_sequences();
+    exec, define_nominal_beams();
+    set, format = '.15e';
+    call, file = 'tests/inputs/models/inj_beam{beam}/opticsfile.1'; !@modifier
+
+    select, flag = twiss, pattern = 'BPM.*B[12]$', column = name, s, {PHASE_ADV}x, {PHASE_ADV}y;
+    use, sequence = LHCB{beam};
+
+    ! Match the tunes initially
+    match, deltap = {offset};
+    vary, name=dQx.b{beam};
+    vary, name=dQy.b{beam};
+    constraint, range = '#E', mux = {Qx}, muy = {Qy};
+    lmdif, tolerance = 1.0e-10;
+    endmatch;
+
+    ! Run a twiss with the offset to get orbit
+    twiss, deltap = {offset};
+
+    ! Correct the orbit
+    correct, mode = svd;
+
+    ! Match the tunes back to normal
+    match, deltap = {offset};
+    vary, name=dQx.b{beam};
+    vary, name=dQy.b{beam};
+    constraint, range = '#E', mux = {Qx}, muy = {Qy};
+    lmdif, tolerance = 1.0e-10;
+    endmatch;
+
+    ! Run the final twiss to get the off-orbit response
+    twiss, deltap = {offset}, file = '{tmp_path}/twiss_dpp_{offset:.1e}_B{beam}.dat';
```

Comment:
Cool

Reply:
Actually, no. The `twiss-` files are all `twiss_xxxx.dat` because historical reasons. We only changed everything to .tfs in the results, but not in the model (to also be backwards compatible with old models). Sorry for the back and forth, Josh